### DISCUSSION TITLE
AKHQ service down after some time running

### DISCUSSION THREAD

**Discussion by gerardgorrion** [tchiotludo/akhq#929] [General]
Hi all,

Our current akhq (helm version chart 0.1.3 and akhq 0.17.0) is deployed into kubernetes in AWS (EKS).  All i working fine, connect with kafka, ldap connections are working, etc etc. But some times, after a lot of days (sometimes 100+ days, sometimes only 20), connections via web fail and we need to kill pod manually.

There are some way to check if the akhq pod was toast and/or some way to recover the service? We think in deploy in HA, but is the same problem but delaying the problem in time (all pods will fails after dsome time).

We serve akhq like a monitoring service to differents teams, and we are very happy with this product, good job!

**Comment by tchiotludo** [comment]
Hi, 

Maybe you share more detail on the pod status ? 
- Is this healthcheck that failed?
- Can you provide log when the pod failed?
- Can you try to connect inside the pods and reach healthcheck ? 

And don't forgot [this one](https://github.com/tchiotludo/akhq/discussions/161) if you are happy with akhq :+1:

**Reply by gerardgorrion** [reply]
I think the problem is the live trail:

When I use a replica:
 - I open two akhq tabs in chrome
 - In the first tab, I run the live trail
 - In the second tab I get a 502 response
 
When I use two replicas:
 - I open two akhq tabs in chrome
 - In the first tab, I run the live trail
 - In the second tab everything is fine
 - One of the 2 pods enters in "Running 0/1" state - no logs are displayed.
 
I have checked the memory of the pod and it seems that it had reached the limit of the pod (4gb)

**Comment by tchiotludo** [comment]
Please update to last version, there is a fix about live tail
